core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -55,7 +55,7 @@ angular.module('oppia').directive('adminMiscTab', [
 
         ctrl.MAX_USERNAME_LENGTH = MAX_USERNAME_LENGTH;
 
-        ctrl.clearSearchIndex = function() {
+        ctrl.clearSearchIndexAsync = async function() {
```

Comment:
@nithusha21, Can you please take the first pass on this PR? I don't have enough context on PR expectations.

Reply:
@nithusha21, any update on this thread?

Reply:
This funtion does not return a promise, so we don't really need the async keyword + Async in the definition.

Reply:
Confirming that we don't need any changes in this file. At the time this file was mentioned in #10306, we had the following method that required this change, however, that method does not exist anymore.

https://github.com/kevintab95/oppia/blob/ed83661de15f33a992d622ea088efad3f2cb16b6/core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts#L204-L278